### PR TITLE
Pin to Calico-owned scapy image.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

Image we were using before has been taken down.

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by tomastigera — APPROVED** [review]
Cool!

**Pull request merged**
